### PR TITLE
Add tooltip component

### PR DESCRIPTION
**Pull request by kevinch**
screen shot 2018-03-02 at 14 49 57

### CODE REVIEW

**Comment by cmdalbem** [comment]
Nice addition @kevinch!
I see that you exposed all the libs properties in our wrapper. I think it'd be interesting to limit a little our developers' freedom here, for the sake of usage simplicity and consistency across the Styleguide uses.
What do you think?

**Comment by kevinch** [comment]
Sure thing, we can discuss this the week.

File: src/components/ToolTip/index.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+import { Tooltip } from 'react-tippy'
+import 'react-tippy/dist/tippy.css'
+import './index.css'
```
Comment (klzns) [inline src/components/ToolTip/index.js]: Isso nao vai funcionar.

File: src/components/ToolTip/index.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+import { Tooltip } from 'react-tippy'
+import 'react-tippy/dist/tippy.css'
```
Comment (klzns) [inline src/components/ToolTip/index.js]: Import de qualquer coisa que nao seja JS ou JSON nao funciona em Node.

**Pull request closed without merging**